### DISCUSSION TITLE
plot count

### DISCUSSION THREAD

**Discussion by alireza-pmahdavi** [Chia-Network/chia-blockchain#3001] [Q&A]
Hello,

I couldn't understand the plot count.
I have 1 TB ssd drive and 7x11TB HDD 
What plot count should I choose?
kX32 with 29 plots?
CPU mode: dual xeon e5-2620
Thanks

**Comment by Keriwaza** [comment]
The plot count determines how many plots you are trying to make at once, that's all. Totally up to you what you decide.
